runelite-client/src/main/resources/net/runelite/client/plugins/gpu/frag.glsl

```diff
@@ -39,12 +40,17 @@ in float fogAmount;
 out vec4 FragColor;
 
 #include hsl_to_rgb.glsl
+#include colorblind.glsl
 
 void main() {
   int hsl = int(fHsl);
   vec3 rgb = hslToRgb(hsl) * smoothBanding + Color.rgb * (1.f - smoothBanding);
   vec4 smoothColor = vec4(rgb, Color.a);
 
+  if (colorBlindMode > 0) {
```

Comment:
Any reason you've decided to apply this prior to the texture sampling? I would think it would be better to apply it after.

Reply:
You're completely right, I've now pushed a fix for that.